p2p/adapters/types.go

```diff
@@ -85,3 +92,23 @@ type Reporter interface {
 	DidConnect(*NodeId, *NodeId) error
 	DidDisconnect(*NodeId, *NodeId) error
 }
+
+
+func RandomNodeId() *NodeId {
```

Comment:
Because testing includes simulations includes testing, so cant use from sims. And they are used to return adapters.NodeIds anyway. :)